tests/test_interface.rb

```diff
@@ -253,7 +295,11 @@ def validate_ipv4_proxy_arp(inttype_h)
       cmd = show_cmd(interface.name)
 
       # puts "value - #{v[:proxy_arp]}"
-      pattern = (/^\s+ip proxy-arp/)
+      if node.client.api == 'NXAPI'
+        pattern = /^\s+ip proxy-arp/
+      elsif node.client.api == 'gRPC'
+        pattern = /^\s+proxy-arp/
```

Comment:
We should start grabbing CLI patterns from YAML instead of manually setting them.  gRPC and NXAPI have different enough CLI's that more cases like this will clutter up our codebase.